scim/resource_entitlement.go

```diff
@@ -28,7 +28,7 @@ func ResourceEntitlements() *schema.Resource {
 	addEntitlementsToSchema(&entitlementSchema)
 	return common.Resource{
 		Create: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
-			return patchEntitlements(ctx, d, c, "add")
+			return patchEntitlements(ctx, d, c, "replace")
```

Comment:
My concern here is that we initialize entitlements with `false`, and when we start to replace them instead of adding, then people may lose the access?

Reply:
this resource is meant to enforce entitlement of said identity - the update function will enforce the entitlements anyway, so this change is to make it consistent